products/BellHybrid/apps/common/include/common/models/RelaxationFadeModel.hpp

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2017-2024, Mudita Sp. z.o.o. All rights reserved.
+// For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
+
+#pragma once
+
+#include "AbstractRelaxationFadeModel.hpp"
+#include "SettingsModel.hpp"
+#include <ApplicationCommon.hpp>
+
+namespace app
+{
+    class RelaxationFadeSettings : public gui::SettingsModel<bool>
+    {
+      public:
+        using SettingsModel::SettingsModel;
+
+        auto setValue(bool value) -> void override;
+        auto getValue() const -> bool override;
+    };
+
+    class RelaxationFadeModel : public AbstractRelaxationFadeModel
+    {
+      public:
+        RelaxationFadeModel() = delete;
+
+        RelaxationFadeModel(ApplicationCommon *app)
+        {
+            fade = std::make_unique<RelaxationFadeSettings>(app);
+        }
+
+        gui::AbstractSettingsModel<bool> &getFade() override
```

Comment:
```suggestion
        auto getFade() -> gui::AbstractSettingsModel<bool> & override
```

To be consistent with the rest of this file :wink:

Reply:
done